packages/bits/src/services/unit-conversion.service.ts

```diff
@@ -132,8 +127,15 @@ export class UnitConversionService {
         return unitConversionConstants[unit][conversion.order];
     }
 
-    private isValidNumber(value: any): boolean {
-        return !isNaN(parseFloat(value)) && isFinite(parseInt(value, 10));
+    /**
+     * Gets the base value of the converted unit
+     *
+     * @param unit The basic unit used in the conversion
+     *
+     * @returns {string} The first unit in the unitConversionConstant array
```

Comment:
Maybe this to avoid revealing implementation details:

```suggestion
     * @returns {string} The abbreviation for the smallest unit of the provided UnitOption
```